### DISCUSSION TITLE
Is it possible to import files

### DISCUSSION THREAD

**Discussion by tamueller** [spakin/SimpInkScr#106] [General]
Hi,
I would like to do some very simple batch processing - is there a way to import a .png file using SimpleInkscapeScpripting? I couldn't find it in the reference.

**Comment by tamueller** [comment]
Ah, I found it, it's called image. I stumbled upon it after saving something I did to .py and look at the source code.

**Reply by spakin** [reply]
I'm glad you found it!  For the record, `image` is documented in the wiki at [Shape construction → Images](https://github.com/spakin/SimpInkScr/wiki/Shape-construction#images).

That said, I admit that saving to a `.py` file and examining the result is a really convenient way to learn how to implement a desired feature.
